templates/components/streamfield/blocks/accordion.html

```diff
@@ -1,5 +1,29 @@
 {% verbatim %}
-<div class="">
-    {{ self }}
+{% load wagtailcore_tags %}
+
+<div class="container mx-auto my-16 text-black">
+    {% for accordion in accordions %}
+    <details class="max-w-3xl my-12 border-b border-grey-300">
+        {# webkit-details-marker hides the default arrow on safari #}
+        <summary
+            class="flex flex-col my-4 gap-2 hover:cursor-pointer focus-visible:outline-0 [&::-webkit-details-marker]:hidden">
+            <span class="font-semibold text-3xl hover:underline">{{ accordion.title }}</span>
+            {# Show/Hide Button - Extra group focus classes to wrap focus around link instead of full block #}
+            <div
+                class="text-base text-mackerel-300 hover:text-mackerel-200 focus-visible:outline focus-visible:outline-2 focus-visible:outline-mackerel-300 focus-visible:w-fit mb-8">
+                {# We'll use CSS classes that we can target with the details[open] selector #}
+                <div class="show-button flex items-center gap-1">
+                    {% include "icons/chevron.html" with class="w-3.5 h-3.5 chevron-icon"%}
```

Comment:
I couldn't find the icon in the PR or in the codebase, should it be included in the PR?

Reply:
Whoops. Yeah. I forgot to move it over from the project I was testing the changes in. I've just added it to the PR. I'll take a look at the Tailwind approach you recommended.